.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Update the Code Coverage
```

Comment:
This workflow should be renamed to `Publish Test Coverage Report`.

Reply:
Renamed to `Publish Test Coverage Report`